### ISSUE TITLE
q-input in a tab pane doesn't get blur event

### ISSUE THREAD

**Issue by dsl101** [quasarframework/quasar#1971]
### Software version

Quasar: 0.15
OS: Win 10
Browsers: Chrome

### JsFiddle (for Quasar v0.15+ only)
https://codepen.io/anon/pen/pVywKg?editors=1111

### What did you get as the error?
Implementing lazy model and 'save on blur'. Clicking tab 'B' doesn't generate a blur event for the q-input if it had focus, and new input can be lost

### What were you expecting?
The q-input to get a blur event with the changed data

### What steps did you take, to get the error?
In trying to implement 'save on blur', the `@blur` event seemed an obvious solution. But when the input element is part of a tab set, its blur event is lost. I presume this is something to do with the tab activation happening first and unloading the tab pane? Is there a way to ensure underlying blur events are still available?

**Comment by nothingismagick** [comment]
Tracking this issue.

**Comment by pdanpdan** [comment]
In this specific case you can use one of the following approaches:

- if the tabs are linked to routes then you can put guards in beforeRouteLeave
- else (or as an alternative) you can use `@input` instead of `@change` and implement yourself the debouncing

**Comment by dsl101** [comment]
Yes, that's what I've done for no (option 2), but I'm hoping it's a temporary workaround :)
